scripts/gen_tablet_screenshots.py

```
@@ -210,80 +208,79 @@ def make_room(W, H, scale):
-    muw = d.textlength("Mute", font=muf)
-    d.text((mx - int(muw) // 2, my - s(16, scale)), "Mute", font=muf, fill=WHITE)
+    # Peers card — linear list matching the actual PeerRow-based UI
+    peers = [
+        ("TR", "Taylor Rivera", PURPLE, True,  False),
+        ("MK", "Morgan Kim",    GREEN,  False, True),
+    ]
+    row_h = s(120, scale)
+    card_total = len(peers) * row_h
+    rounded_rect(d, [PAD, y, W - PAD, y + card_total], r=s(16, scale), fill=CARD_BG)
+
+    for i, (initials, name, col, talking, muted) in enumerate(peers):
```

Ruff (0.15.12)

[warning] Loop control variable `muted` not used within loop body

Rename unused `muted` to `_muted`

(B007)